modules/nf-core/salmon/quant/main.nf

```diff
@@ -64,6 +65,10 @@ process SALMON_QUANT {
         $args \\
         -o $prefix
 
+    if [ -f $prefix/aux_info/meta_info.json ]; then
+        cp $prefix/aux_info/meta_info.json .
```

Comment:
I may be useful to also incorporate prefix in this filename for distinction purposes?

Reply:
Excellent suggestion!